exonum/src/blockchain/config.rs

```diff
@@ -295,6 +296,131 @@ impl ValidateInput for ConsensusConfig {
     }
 }
 
+#[derive(
+    Debug,
+    Clone,
+    PartialEq,
+    Eq,
+    Hash,
+    Serialize,
+    Deserialize,
+    ProtobufConvert,
+    BinaryValue,
+    ObjectHash,
+)]
+#[protobuf_convert(source = "runtime::GenesisConfig")]
+pub struct GenesisConfig {
+    /// Blockchain configuration used to create the genesis block.
+    pub consensus_config: ConsensusConfig,
+
+    /// Artifacts specification of builtin services.
+    pub artifacts: Vec<ArtifactSpec>, // TODO: HashSet? Duplications? Perhaps check in builder?
+
+    /// List of the privileged services with the configuration parameters that are created directly
```

Comment:
Why are they priveleged?

Reply:
I took as-is from the `BlockchainBuilder` 😌. I guess the privilege is that they're deployed and started without corresponding transactions. Perhaps this could be reworded as "List of the services with its configuration parameters that are created directly in the genesis block".

Reply:
At the moment "privileged" is obsolete term in documentation. We have switched to the different approach where only "supervisor" service is privileged.